src/services/identity/UsersService.ts

```diff
@@ -1,5 +1,5 @@
 import { ResultAsync, errAsync, okAsync } from "neverthrow"
```

Comment:
Could we share the before and after trace in the pr description to see the difference and also maybe if the latency improvements

Reply:
> Could we share the before and after trace in the pr description

The after traces are already there, I'll add a before trace as well.


> if the latency improvements

Aye, so on this a few things:
* The current query currently runs under 2ms most of the time, so the optimization isn't all that interesting from a latency perspective to be honest, and we are not expecting a big performance boost or anything
* A dev pattern of downloading the full table content is bad generally, so the PR serves as illustration of how more complex query that are not necessarily supported by the ORM. The ORM is a tool, it should not be a blocker to run inefficient queries.
* I already added in the IsomerCMS dashboard a block to monitor the performance of the old query versus the new query for comparison [here](https://ogp.datadoghq.com/dashboard/8tj-6k5-p2v?fromUser=false&refresh_mode=paused&view=spans&from_ts=1713230653828&to_ts=1713245053828&live=false&tile_focus=7015368340178734)

Reply:
> Could we share the before and after trace in the pr description

Done! 😄  I added a sample trace for the before in the PR description and I updated the performance section too. 

So again, in the grand scheme of things, this is a very cheap query so this PR is not bringing visible impact to users. It's more to illustrate how to query the DB with good patterns :) 

Also Just for reference, here is a screenshot on the variability of max run time of the current query in prod. We'll be comparing the performance of the new query against that ([link to widget here](https://ogp.datadoghq.com/dashboard/8tj-6k5-p2v/isomer-cm-s?fromUser=false&refresh_mode=paused&view=spans&from_ts=1713089089262&to_ts=1713261889262&live=false&tile_focus=3522012591074081))

![image](https://github.com/isomerpages/isomercms-backend/assets/935223/57132d04-0000-422d-89ac-ce37c11a8c3b)